src/main/scala/strawman/collection/Iterator.scala

```diff
@@ -127,6 +159,34 @@ trait Iterator[+A] extends IterableOnce[A] { self =>
     this
   }
 
+  /*
+   * Implemented by means of a buffer to keep the last n elements from being returned during iteration.
+   */
+  def dropRight(n: Int): Iterator[A] = {
```

Comment:
Yes, that’s what I meant: we provide this operation on `Iterable` at the cost of two traversals.